lib/matplotlib/artist.py

```diff
@@ -1053,6 +1054,35 @@ def set_in_layout(self, in_layout):
         """
         self._in_layout = in_layout
 
+    def get_aria(self):
+        """Return any ARIA properties assigned to the artist"""
+        return dict(self._aria)
+
+    def set_aria(self, aria):
+        """
+        Set ARIA properties to the artist.
+
+        A primary use of this method is to attach aria-label to the artist to
+        provide alt text to figures.
+
+        Parameters
+        ----------
+        aria : dict
+
+        """
+        # TODO validation
+        if not isinstance(aria, dict):
+            if aria is not None:
+                raise TypeError(
+                    f'aria must be dict or None, not {type(aria)}')
+        self._aria = aria
+
+    def update_aria(self, **aria):
```

Comment:
Do we want `update_aria` or only get/set like most of our other methods.

I think a user could then do something like: `a.set_aria(a.get_aria() | new_aria_dict)`